jdaviz/app.py

```diff
@@ -1167,6 +1167,7 @@ def _get_first_viewer_reference_name(
         )
 
         spectral_viewers = (SpecvizProfileView, CubevizProfileView)
+        spectral_2d_viewers = (MosvizProfile2DView, )
         table_viewers = (MosvizTableViewer, )
         image_viewers = (MosvizProfile2DView, CubevizImageView, SpectralExtraction)
```

Comment:
I would expect this to have been `(ImvizImageView, CubevizImageView)`... are `MosvizProfile2DView` and/or `SpectralExtraction` necessary for any existing plugins to function (@bmorris3)?

Reply:
I can't be sure without experimenting – have you tried on this branch??

Reply:
`SpectralExtraction` is a plugin, it must have been added for a reason?  I can try and run tests and just make sure plugins don't crash on load, but it's hard to tell if it has other unintended consequences (the slit overlay bug slipped under the radar for quite a while) 😬

Reply:
passes tests and couldn't find any issues when manually testing, but further testing during reviews is welcomed!